src/components/plots/helpers/dot-plot/getDotPlotData.js

```diff
@@ -17,29 +19,37 @@ const getClusterNames = (state) => {
   return clusterNames;
 };
 
-const fetchPlotDataWork = (
+const getDotPlotData = (
```

Comment:
I decided to create separate functions for each of the plot work data calls because it's getting difficult and absurd to have to pass extra parameters needed by only one plot